app/views/spree/admin/overview/_version.haml

```diff
@@ -0,0 +1,3 @@
+%a{href:"https://github.com/openfoodfoundation/openfoodnetwork/releases", target: "_blank", title: t('.view_all_releases')}
+  =# Show the latest tag. If there are commits since the tag, show number of commits and an identifier. If the working tree is dirty, show 'modified'.
+  = `git describe --tags --dirty=-modified`
```

Comment:
oh wahoo. I didn't know this was actually possible: running git in our view!
I don't have any better suggestion for now (maybe put in a controller ?) as it seems that we don't package a release.

Reply:
Yes, it does feel a bit wrong.. but I think to change it would just be more work and obfuscate it further without any benefit.